CryptoPrices/Sources/MyCoin/Sources/MyCoin/MyCoinView/MyCoinView.swift

```diff
@@ -70,9 +70,7 @@ private extension MyCoinView {
     var coinStatisticsSection: some View {
         Section {
             CoinStatisticsSection(
-                coinDetailItem: CoinDetailItem(
-                    coinDetail: MockCoinDetail.single
-                )
+                coinDetailItem: viewModel.coinDetail
             )
         }
     }
```

Comment:
Could you try to keep `CoinStatisticsSection` unchanged (everything is still non `Optional`, no `if let`, no `?? false`, no `noData` state ...), and just update like this for `MyCoinView`:
```swift
var coinStatisticsSection: some View {
  viewModel.coinDetail.map { coinDetail in
    Section {
      CoinStatisticsSection(coinDetailItem: coinDetail)
    }
  }
}
```

Reply:
Thank you for the suggestion 🙌 . It works correctly when showing an empty view if real data is nil. But I still wanted to keep the structure of the statistics section with "-" for data that are not found as the following. What do you think?
<img width="300" src="https://user-images.githubusercontent.com/25881847/211994776-7fbed2ea-c3d1-4c10-bbef-68b8391511af.png">

Reply:
@nkhanh44 If that's what you expect, it's fine to go with your approach 👍 (Still to be honest, it hurts me when seeing a lot of nil coalescing (`?? false`) and imperative codes (`if let else`) in declarative SwiftUI views 😄)

One more thing, could you help to adjust some vertical spacings: between the `coinStatisticsItemView`s and the inner of the `coinStatisticsItemView` itself 🙏

![Simulator Screen Shot - iPhone 14 - 2023-01-12 at 14 56 05](https://user-images.githubusercontent.com/17972674/212010383-60186df4-4da5-4636-8803-f558c6d12269.png)

From the screenshot of our app, the (inner) spacing between `"Market Cap"` and `"$168,926,..."` is bigger than that of between `"$168,926,..."` and `"All Time High"` (outer one). So visually it's hard to see the clear separation between those groups, it even can make us confused about which group `"$168,926,..."` should it belong to.

The design looks good though:
<img width="343" alt="Screenshot 2023-01-12 at 14 56 24" src="https://user-images.githubusercontent.com/17972674/212010143-e9dc94a8-411a-480f-a5b4-0afdb40f70df.png">

Reply:
@Thieurom Thank you for the raise, I agree with your concern 👍. I refactored the code in the coin's statistic without a lot of nil coalescing (?? false) and imperative codes (if let else). Please help me to review if it's fine with you 🙏 .